src/preamble.lua

```diff
@@ -5,11 +5,34 @@ function Enum.__tostring(t)
 end
 
 function Enum.new(k, v)
-  local e = {k, v}
+  local e = { k, v }
```

Comment:
Agressive

Reply:
I would blame it on my formatter, but that was probably me ;)